docs/admin/external-auth.md

```diff
@@ -27,62 +33,62 @@ The `CODER_EXTERNAL_AUTH_0_ID` environment variable is used for internal
 reference. Therefore, it can be set arbitrarily (e.g., `primary-github` for your
 GitHub provider).
 
-## GitHub
+Add the following code to any template to add a button to the workspace setup page which will allow you to authenticate with your provider:
 
-> If you don't require fine-grained access control, it's easier to configure a
-> GitHub OAuth app!
+```tf
+data "coder_external_auth" "<github|gitlab|azure-devops|bitbucket-cloud|bitbucket-server|etc>" {
+    id = "<USER_DEFINED_ID>"
+}
 
-1. [Create a GitHub App](https://docs.github.com/en/apps/creating-github-apps/registering-a-github-app/registering-a-github-app)
+# Github Example (CODER_EXTERNAL_AUTH_0_ID="github-auth")
+# makes a github authentication token available at data.coder_external_auth.github.access_token
```

Comment:
Is the preceding `#` expected in this line and the above?

Wouldn't this make it a level 1 heading?

Reply:
this is part of the `tf ` code block, so this should just be a comment

Reply:
Ooops.